ui/raidboss/data/06-ew/dungeon/lapis_manalis.txt

```diff
@@ -0,0 +1,283 @@
+# Lapis Manalis
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 00:0839::.*is no longer sealed/ window 100000 jump 0
+
+#~~~~~~~~#
+# Albion #
+#~~~~~~~~#
+
+# -ii 7A7D
+
+0.0 "--sync--" sync / 00:0839::The Silvan Throne will be sealed off/ window 0,1
+6.0 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+10.6 "Call of the Mountain" sync / 1[56]:[^:]*:Albion:7A7C:/
+21.6 "Wildlife Crossing 1" sync / 00:0839::A rumbling in the distance draws closer.../ window 30,30 duration 15
+35.6 "Wildlife Crossing 2" sync / 00:0839::More rumbling can be heard in the distance.../ window 30,30 duration 15
+39.2 "Albion's Embrace" sync / 1[56]:[^:]*:Albion:7A85:/
+45.3 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+51.7 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
+60.3 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7E:/
+62.3 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7F:/
+68.6 "Icebreaker" sync / 1[56]:[^:]*:Albion:7A81:/
+71.6 "Icy Throes" sync / 1[56]:[^:]*:Albion:7A82:/
+77.8 "Icy Throes" sync / 1[56]:[^:]*:Albion:(800F|7A83):/
+78.2 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+82.8 "Icy Throes" sync / 1[56]:[^:]*:Albion:7FB9:/
+86.8 "Roar of Albion" sync / 1[56]:[^:]*:Albion:7A84:/
+99.9 "Call of the Mountain" sync / 1[56]:[^:]*:Albion:7A7C:/
+105.3 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+110.9 "Wildlife Crossing 1" sync / 00:0839::A rumbling in the distance draws closer.../ window 30,30 duration 15
+112.1 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
+119.6 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+124.9 "Wildlife Crossing 2" sync / 00:0839::More rumbling can be heard in the distance.../ window 30,30 duration 15
+125.9 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
+135.1 "Albion's Embrace" sync / 1[56]:[^:]*:Albion:7A85:/
+143.1 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7E:/
+145.1 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7F:/
+151.2 "Icebreaker" sync / 1[56]:[^:]*:Albion:7A81:/
+154.2 "Icy Throes" sync / 1[56]:[^:]*:Albion:7A82:/
+160.4 "Icy Throes" sync / 1[56]:[^:]*:Albion:(800F|7A83):/
+160.8 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+165.4 "Icy Throes" sync / 1[56]:[^:]*:Albion:7FB9:/
+169.4 "Roar of Albion" sync / 1[56]:[^:]*:Albion:7A84:/ window 30,30 jump 86.8
+182.6 "Call of the Mountain" sync / 1[56]:[^:]*:Albion:7A7C:/
+188.0 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+193.4 "Wildlife Crossing 1" sync / 00:0839::A rumbling in the distance draws closer.../ window 30,30 duration 15
+194.6 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
+202.2 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+207.2 "Wildlife Crossing 2" sync / 00:0839::More rumbling can be heard in the distance.../ window 30,30 duration 15
+208.3 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
```

Comment:
For timelines, can you call this `Left Slam/Right Slam` (full ability names, for making translations easier) and then use a `timelimeReplace` for English, e.g. https://github.com/quisquous/cactbot/blob/main/ui/raidboss/data/06-ew/raid/p8s.ts#L2722-L2740